tensorflow/lite/micro/examples/network_tester/network_tester_test.cc

```diff
@@ -17,15 +17,21 @@ limitations under the License.
 #include "tensorflow/lite/micro/examples/network_tester/expected_output_data.h"
 #include "tensorflow/lite/micro/examples/network_tester/input_data.h"
 #include "tensorflow/lite/micro/examples/network_tester/network_model.h"
+#include "tensorflow/lite/micro/examples/person_detection/person_detect_model_data.h"
+#include "tensorflow/lite/micro/examples/person_detection/person_image_data.h"
```

Comment:
should this be behind an ifdef ETHOS_U?

Reply:
Yes let's do so for the sake of consistency.